kotlin/services/elasticbeanstalk/src/main/kotlin/com/aws/example/DescribeEnvironment.kt

```diff
@@ -0,0 +1,64 @@
+//snippet-sourcedescription:[DescribeEnvironment.kt demonstrates how to describe an AWS Elastic Beanstalk environment.]
+//snippet-keyword:[SDK for Kotlin]
+//snippet-keyword:[Code Sample]
+//snippet-service:[AWS Elastic Beanstalk ]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[03/10/2022]
+//snippet-sourceauthor:[scmacdon - aws]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.aws.example
+
+//snippet-start:[eb.kotlin.describe_env.import]
+import aws.sdk.kotlin.services.elasticbeanstalk.ElasticBeanstalkClient
+import aws.sdk.kotlin.services.elasticbeanstalk.model.DescribeEnvironmentsRequest
+import kotlin.system.exitProcess
+//snippet-end:[eb.kotlin.describe_env.import]
+
+/**
+To run this Kotlin code example, ensure that you have set up your development environment,
+including your credentials.
+
+For information, see this documentation topic:

```

Comment:
For more information, see the following documentation topic:

Reply:
fixed